firstvoices/backend/tests/test_tasks/test_build_mtd.py

```diff
@@ -158,3 +164,115 @@ def test_old_results_removed(self, site):
         assert len(saved_results) == 1
         assert saved_results.latest().latest_export_result == final_result
         assert not saved_results.latest().is_preview
+
+
+class TestCheckSitesForMTDSyncTask:
+    @pytest.fixture(scope="function", autouse=True)
+    def mocked_mtd_build_func(self, mocker):
+        self.mocked_func = mocker.patch(
+            "backend.tasks.build_mtd_export_format_tasks.build_index_and_calculate_scores.apply_async"
+        )
+
+    @pytest.fixture(autouse=True)
+    def sites(self):
+        return {
+            "site_one": factories.SiteFactory.create(slug="site_one"),
+            "site_two": factories.SiteFactory.create(slug="site_two"),
+            "site_three": factories.SiteFactory.create(slug="site_three"),
+        }
+
+    @pytest.mark.django_db
+    def test_no_sites_eligible_to_sync(self):
+        result = check_sites_for_mtd_sync.apply()
```

Comment:
Please add another case where only private sites have changes, and we don't queue the task